pkg/sentry/fsimpl/fuse/dev.go

```diff
@@ -302,9 +303,11 @@ func (fd *DeviceFD) noReceiverAction(ctx context.Context, r *Response) error {
 	delete(fd.requestKind, r.hdr.Unique)
 
 	if opCode == linux.FUSE_INIT {
-		// TODO: process init response here.
-		// Maybe get the creds from the context?
-		// creds := auth.CredentialsFromContext(ctx)
+		return fd.fs.InitRecv(
+			auth.CredentialsFromContext(ctx),
```

Comment:
I wonder if the ctx is the same as the one from InitSend? I believe we should use the ctx of InitSend

Reply:
I think here we can use the context of the fuse daemon here. I think the Linux does the same thing.

Reply:
By "fuse daemon", you mean whatever kernel task we use to poll the reply?

Reply:
I mean a user-space fuse daemon which writes the reply. noReceiverAction is called from DeviceFD.writeLocked, so ctx here is a context of a task which writes in the file descriptor.

I think we can remove this TODO, because ctx points to the right context.